packages/cosmic-swingset/lib/ag-solo/vats/lib-wallet.js

```diff
@@ -1,22 +1,30 @@
 import harden from '@agoric/harden';
 import { insist } from '@agoric/ertp/util/insist';
 
+import { makeUnitOps } from '@agoric/ertp/core/unitOps';
+
+import makeObservablePurse from './observable';
+import makeStore from './store';
+import makeWeakStore from './weakStore';
+
 function mockStateChangeHandler(_newState) {
```

Comment:
I realize this isn't changing in this PR.

MockFoo means something specific in test frameworks. Can this be NullStateChangeHandler, or NoActionStateChangeHandler instead?

Reply:
yeah, good point